Source/JavaScriptCore/jit/BaselineJITCode.cpp

```diff
@@ -53,8 +53,17 @@ BaselineJITCode::BaselineJITCode(CodeRef<JSEntryPtrTag> code, CodePtr<JSEntryPtr
     , MathICHolder()
 { }
 
-BaselineJITCode::~BaselineJITCode()
+BaselineJITCode::~BaselineJITCode() = default;
+
+CodeLocationLabel<JSInternalPtrTag> BaselineJITCode::getCallLinkDoneLocationForBytecodeIndex(BytecodeIndex bytecodeIndex) const
 {
+    auto* result = binarySearch<const BaselineUnlinkedCallLinkInfo, BytecodeIndex>(m_unlinkedCalls.data(), m_unlinkedCalls.size(), bytecodeIndex,
```

Comment:
This doesn't need a lock?

Reply:
This is not necessary since we set up this array at JITCode creation and after that, we never changes.